R/create_demographic_lookup.R

```diff
@@ -0,0 +1,392 @@
+#' Create the demographic lookup file
+#'
+#' @param data A data frame with the required variables
+#' @param year The year in standard SLF format
+#' @param write_to_disk Defaults to TRUE
+#'
+#' @export
+#'
+#' @family Demographic and Service Use Cohort functions
+create_demographic_lookup <- function(data, year, write_to_disk = TRUE) {
```

Comment:
```suggestion
create_demographic_cohort <- function(data, year, write_to_disk = TRUE) {
```
One small name change suggestion as i just noticed this and we might get confused with the social care demographic lookup

Reply:
```suggestion
create_demographic_cohorts <- function(data, year, write_to_disk = TRUE) {
```